### PR TITLE
MW Proofreading edits

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

Staging - https://preview-mongodbrustagir.gatsbyjs.io/rust/MW-proofreading-pre-publiush/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/fundamentals/crud/compound-operations.txt [hunk 1]
```diff
@@ -112,34 +112,35 @@ The following table describes the options available in
...
          in the Server manual glossary.
        | Type: ``Document``
-       | Default: ``nil``
+       | Default: ``None``
 
    * - ``write_concern``
-     - | The write concern for the operation. To learn more about write
-         concerns, see :manual:`Write Concern
+     - | The write concern for the operation. If you don't set a write
+         concern, the operation inherits the one set for the collection.
```
Comment (norareidy) [inline source/fundamentals/crud/compound-operations.txt:127]: S: the use of "one" is a little unclear, so could reword like this

```suggestion
     - | The write concern for the operation. If you don't set this option, the operation inherits the write concern set for the collection.
```

File: source/fundamentals/crud/compound-operations.txt [hunk 2]
```diff
@@ -233,7 +235,7 @@ The following table describes the options available in
 
    * - ``array_filters`` 
      - | The set of filters specifying the array elements to which the
-         update should apply.
+         update applies to.
```
Comment (norareidy) [inline source/fundamentals/crud/compound-operations.txt:238]:
```suggestion
         update applies.
```

File: source/fundamentals/crud/compound-operations.txt [hunk 3]
```diff
@@ -275,23 +277,24 @@ The following table describes the options available in
        | Default: ``false``
 
    * - ``write_concern``
-     - | The write concern for the operation. To learn more about write
-         concerns, see :manual:`Write Concern
+     - | The write concern for the operation. If you don't set a write
+         concern, the operation inherits the one set for the collection.
+         To learn more about write concerns, see :manual:`Write Concern
```
Comment (norareidy) [inline source/fundamentals/crud/compound-operations.txt:282]: S: same as above

```suggestion
     - | The write concern for the operation. If you don't set this option, the operation inherits the write concern set for the collection.
         To learn more about write concerns, see :manual:`Write Concern
```

Comment (norareidy) [reply source/fundamentals/crud/compound-operations.txt:282]: Applies elsewhere too

File: source/fundamentals/crud/read-operations/query.txt [hunk 4]
```diff
@@ -186,10 +186,10 @@ Query Operators </reference/operator/query-comparison/>` in the Server manual.
 Logical
 -------
 
-Logical operators require at least two match criteria and match documents based on
-if they meet some, all, or none of these criteria. For example, you can use the ``$or``
-method to query for documents that match either a ``$gt`` comparison
-operator or a literal value query. 
+Logical operators require at least two match criteria and can match
+documents that meet some, all, or none of these criteria. For example,
+you can use the ``$or`` method to query for documents that match either
```
Comment (norareidy) [inline source/fundamentals/crud/read-operations/query.txt:191]:
```suggestion
you can use the logical operator ``$or`` to query for documents that match either
```

File: source/fundamentals/crud/write-operations/change.txt [hunk 5]
```diff
@@ -358,7 +358,7 @@ The following table describes the options available in ``UpdateOptions``:
 
    * - ``array_filters``
      - | Filters specifying to which array elements an update
-         should apply to.
+         applies to.
```
Comment (norareidy) [inline source/fundamentals/crud/write-operations/change.txt:361]:
```suggestion
         applies.
```

**Review by norareidy — APPROVED** [review]
LGTM with a few suggestions!

**Comment by rustagir** [comment]
oops accidentally closed

**Pull request merged**